fragdenstaat_de/fds_newsletter/utils.py

```diff
@@ -206,19 +206,47 @@ def get_onboarding_subscribers(date: datetime.date, schedule_item):
     )
 
 
+def check_csv_format(csv_data):
+    csv_format = {}
+    headers = csv_data.fieldnames
+
+    # TODO: find better place for csv format definitions
+    pretix = ["Bestellnummer", "E-Mail", "Anfragedatum", "Name", "Vorname", "Nachname"]
+
+    if headers == pretix:
+        csv_format = {
+            "name": "Name",
+            "email": "E-Mail",
+        }
+    elif "first_name" in headers and "last_name" in headers:
+        csv_format = {
+            "name": "combine",
```

Comment:
Not a fan of storing different things in `name`: both a column name and a magic constant which implies that two columns should be combined. I suggest to instead return a list of fields here and checking in `import_csv` whether a list was returned (`isinstance(row['name'], list)`)

Reply:
To simplify this could also always return a list for `name`, which for the other cases only contains a single entry